test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -24,15 +40,16 @@ void LoadBalancerFuzzBase::initializeASingleHostSet(
   uint32_t hosts_made = 0;
   // Cap each host set at 256 hosts for efficiency - Leave port clause in for future changes
   while (hosts_made < std::min(num_hosts_in_priority_level, MaxNumHostsPerPriorityLevel) &&
-         port < 65535) {
-    host_set.hosts_.push_back(makeTestHost(info_, "tcp://127.0.0.1:" + std::to_string(port)));
+         port < 60000) {
```

Comment:
update the comment above.

Reply:
Switched to: "// Cap each host set at 60000 hosts - however, let port number enforce a max of 60k hosts across
  // all priority levels."